packages/elements/src/components/ino-menu/ino-menu.scss

```diff
@@ -4,4 +4,13 @@
 
 ino-menu {
   display: block;
+  border-radius: 4px;
```

Comment:
The `4px` overwrites the value I provide. It does not change the border-radius with the following code:
```css
ino-menu {
  border-radius: 0;
}
```

Reply:
You should now be able to set it to a different value. I removed the property from the stylesheet.

Reply:
Ok, my bad here.
I thought that it behaves dynamically e.g. I can change the `border-radius` to `99px` in the DEV-Tools and it updates instantly. Turns out that it's static and was working before and I did not use it correctly.
I'll revert your last commit and approve it!

Reply:
Oh good to know! 😄 Thanks for the heads up!